controllers/ovndbcluster_controller.go

```diff
@@ -719,7 +719,9 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	// cluster member so it can be resolved from outside cluster (edpm nodes)
 	if instance.Spec.NetworkAttachment != "" {
 		var dnsIPsList []string
-		for _, ovnPod := range podList.Items[:*(instance.Spec.Replicas)] {
+		// TODO(averdagu): change to built in min once go1.21 is used
+		min_len := ovn_common.Get_min(len(podList.Items), int(*(instance.Spec.Replicas)))
```

Comment:
don't use snake-names. should be `minLen`. But actually, probably a better name can be used, e.g. `numActiveMemberPods` or smth else more explicit.